docs/cpp_libs_versions.md

```diff
@@ -0,0 +1,21 @@
+## build packages
+* make=3.82
+* cmake=2.8.12.2
+* curl=7.29.0
+* git=1.8.3
+* gcc-c++=4.8.5
+ 
+## rabbitmq and redis
+* rabbitmq-server=3.7.16
+* redis-server=5.0.5
+ 
+## C++ libraries
+* libcurl=7.29.0
+* libcurl-devel=7.29.0
+* boost=1.66.0
+* yaml-cpp=0.5.3
+* rabbitmq-c=0.8.0
+* SimpleAmqpClient=2.4.0
+* cfitsio=3.45
+* hiredis=0.14.0
+* daq-sdk=2.7
```

Comment:
How can me sure that the package in the documentation are the ones spelled out in the bash scripts?

Reply:
I have put the version numbers of tar balls in the filename.